.github/workflows/sanity.yml

```diff
@@ -0,0 +1,85 @@
+name: Sanity
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches:
+      - main
+      - stable-*
+    tags:
+      - '*'
+
+jobs:
+  sanity:
+    uses: ansible-network/github_actions/.github/workflows/sanity.yml@main
+    with:
+      matrix_include: "[]"
+      matrix_exclude: >-
+          [
+            {
+              "ansible-version": "stable-2.9"
+            },
+            {
+              "ansible-version": "stable-2.12",
+              "python-version": "3.7"
+            },
+            {
+              "ansible-version": "stable-2.12",
+              "python-version": "3.11"
+            },
```

Comment:
We shouldn't be testing 2.12 at all. It's no longer supported.

Reply:
I assume defaults are picked up from https://github.com/ansible-network/github_actions/blob/main/.github/workflows/sanity.yml#L19, so I entirely removed any 2.12 combination.

Reply:
Yeah, I believe that should work.